### ISSUE TITLE
Review Etms_Client to correctly handle errors for each function.

### ISSUE THREAD

**Issue by shadowx787** [wvelebanks/CMSC495#14]
Need to do a review over all of the code to handle different possible errors and make sure they are handled correctly.

**Comment by wvelebanks** [comment]
that would include exceptions as well?

**Comment by shadowx787** [comment]
Yes @wvelebanks we need to make sure exceptions are handled correctly as well, I know so far I often catch them and do nothing, so we'll have to go back and fix that if we have time.
